ash/src/entry.rs

```diff
@@ -77,7 +77,12 @@ pub trait EntryV1_0 {
     type Instance;
     fn fp_v1_0(&self) -> &vk::EntryFnV1_0;
     fn static_fn(&self) -> &vk::StaticFn;
-    #[doc = "<https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/vkCreateInstance.html>"]
```

Comment:
Can you change the doc links back? These were originally generated and if we need to re-generate them it will be easier if they all have the same pattern (i.e. all doc attributes).

Reply:
Got those changed back!

Reply:
It currently renders as:

![image](https://user-images.githubusercontent.com/221559/75614519-9ae7c280-5b07-11ea-9089-1f86a784d40b.png)

I think putting it above would be nicer (**note the empty line**):

```
#[doc = "<https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/vkCreateInstance.html>"]
///
/// # Safety
/// In order for the created `Instance` to be valid for the duration of its
/// usage, the `Entry` this was called on must be dropped later than the
/// resulting `Instance`.
```

Will render as:

![image](https://user-images.githubusercontent.com/221559/75614567-177aa100-5b08-11ea-86de-7907f92eec58.png)